package/MDAnalysis/lib/distances.py

```diff
@@ -1636,7 +1636,7 @@ def calc_dihedrals(coords1: Union[np.ndarray, 'AtomGroup'],
         Array containing the dihedral angles formed by each quadruplet of
         coordinates. Values are returned in radians (rad). If four single
         coordinates were supplied, the dihedral angle is returned as a single
-        number instead of an array.
+        number instead of an array. Range of the dihedral angle is (:math:`-\pi`, :math:`\pi`)
```

Comment:
Please stay with in 79 characters linewidth.

You can also make the whole range a math expression
```rest
:math:`(-\pi, \pi)`
```

Reply:
Add an article 
```reST
The range of ...
```